mne/report.py

```diff
@@ -769,6 +769,9 @@ def _add_figs_to_section(self, figs, captions, section='custom',
             warnings.warn('Could not import mayavi. Trying to render '
                           '`mayavi.core.scene.Scene` figure instances'
                           ' will throw an error.')
+        if np.isscalar(scale) and scale < 1.:
+            raise ValueError('scale, if float, must be >= 1 not %s'
+                             % scale)
```

Comment:
I think this would require first figuring out what the minimum is (it's probably somewhere above 0), and then you'd have to inspect every object in every axis of every figure that is passed in to find ones that have text, check their current sizes, and figure out if they'd get scaled to small :(

If @dengemann really needs shrinking, we could just give a warning.